share/functions/__fish_print_xdg_applications_directories.fish

```diff
@@ -0,0 +1,10 @@
+function __fish_print_xdg_applications_directories --description 'Print directories where desktop files are stored'
+    set -l search_path ~/.local/share/applications /usr/share/applications
+    if test -d /usr/local/share/applications
```

Comment:
Why does this directory need an explicit check for existence but not the other two? For example, my ubuntu system has */usr/share/applications* but does not have a *~/.local/share/applications* directory.